db/migrate/20170222194402_copy_agent_id_to_miq_server_id_in_jobs.rb

```diff
@@ -0,0 +1,18 @@
+class CopyAgentIdToMiqServerIdInJobs < ActiveRecord::Migration[5.0]
+  class Job < ActiveRecord::Base
+  end
+
+  def up
+    say_with_time("copying data from agent_id column to miq_server_id column on jobs table") do
+      Job.find_each do |job|
+        job.update_attributes(:miq_server_id => job.agent_id)
+      end
+    end
+  end
+
+  def down
+    say_with_time("nullifying miq_server_id column on jobs table") do
+      Job.update_all(:miq_server_id => nil)
```

Comment:
in ```down``` migration: ```miq_server_id``` updated to the same value - ```nil```
in ```up``` migration:  ```agent_id``` value could be different for different records